docs/DECISIONS.md

```
@@ -479,6 +479,31 @@ The end-to-end browser checklist is captured in the implementation PR. Critical
 
 ---
 
+## ADR-016: Adopt Starwind Pro Blog 6 (Horizontal Cards) on the Homepage
+
+**Date**: 2026-04-28
+**Status**: Accepted
+
+### Context
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)